LukeMason33/Rotten-Tomatillos
# Feature/cypress testing

### Is this a fix or a feature?
- This is a feature that implements some end to end testing using Cypress. We are running these tests using some dummy data that mocks the data coming through from our server. We are testing user functionality through out the sight, as well as testing the the API fetch requests to ensure the requests are coming back as OK responses.


### What is the change/fix?
- The change is the now implemented Cypress directories and files.



### Where should the reviewer start?
- The reviewer should start in the the Cypress directory and look at the Fixtures directory which holds our dummy data, and then look at our integration directory which contains are actual tests.



### How should this be tested?
- This should be tested by pulling down this branch, running `npm i` in the console to install cypress and all other dependencies, then run `npm open cypress` in your console which will then open Cypress and run the tests for this app. Also, ensure that the server is running for the front end.
